study-builder/studies/pancan/emails/survey_reminder.tmpl.html

```diff
@@ -0,0 +1,127 @@
+<html>
+<meta name="format-detection" content="telephone=no,date=no,address=no,email=no,url=no" />
+
```

Comment:
I'm not sure if this email is use for About Cancer, About You, or both. Still waiting on further details, for now, I'm including the template in this PR and calling it `survey_reminder` for now.

Reply:
I've received more information on this and will address in separate PR.